app/posts/manage-your-appointments.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: collection
+title: Manage your appointments
```

Comment:
I'm slightly wary of how generic this service name is. Should stuff be grouped under a `vaccs` namespace? I can see pros and cons of doing so.

Reply:
Yeah... This service _might_ get renamed to something more specific in future.